lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -77,6 +79,11 @@ If ($result.pester_version -ge "4.0.0") {
 if($tags.count){
     $Parameters.Tag = $tags
 }
+
+if($output_file){
```

Comment:
Shouldn't you check `OutputFormat` as well? or add to the param part to fail if both aren't defined together

Reply:
`output_format` has a default value and is only valid if `output_file` is mentioned, hence didn't add any check for it.